independent-publisher-connectors/SOS Inventory/apiProperties.json

```diff
@@ -0,0 +1,38 @@
+{
+	"properties": {
+		"connectionParameters": {
+			"token": {
+				"type": "oauthSetting",
+				"oAuthSettings": {
+					"identityProvider": "oauth2",
+					"clientId": "[[DUMMY]]",
+					"ClientSecret": "[[DUMMY]]",
+					"scopes": [],
+					"redirectUrl": "https://logic-apis-[parameters('location')].consent.azure-apim.net/redirect",
+					"properties": {},
+					"customParameters": {
+						"authorizationUrl": {
+							"value": "https://api.sosinventory.com/oauth2/authorize"
+						},
+						"tokenUrl": {
+							"value": "https://api.sosinventory.com/oauth2/token"
+						},
+						"refreshUrl": {
+							"value": "https://api.sosinventory.com/oauth2/token"
+						}
+					}
+				}
+			}
+		},
+		"capabilities": [],
+		"description": "Retrieves and updates information in SOS Inventory",
+		"displayName": "SOS Inventory",
+		"iconBrandColor": "#da3b01",
+		"publisher": "Harold Anderson",
+    	"stackOwner": "Saddle Oak Software, Inc.",
+		"backendService": {
```

Comment:
@Harold-Anderson please remove this property, it's not required in [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).